en-US/Characters_Codex/06_Spellcasting/Spell_Descriptions/N/Nondetection.md

```diff
@@ -1,12 +1,29 @@
 #### Nondetection
-<!-- TODO Check and tag this spell -->
+<!-- markdownlint-disable link-image-reference-definitions -->
+[_metadata_:spell_name]:- "Nondetection"
+[_metadata_:spell_level]:- "3"
+[_metadata_:spell_school]:- "abjuration"
+[_metadata_:ritual]:- "false"
+[_metadata_:casting_time_amount]:- "1"
+[_metadata_:casting_time_unit]:- "action"
+[_metadata_:range]:- "Touch"
+[_metadata_:target]:- "one willing creature, or a place or object"
+[_metadata_:components_verbal]:- "true"
+[_metadata_:components_somatic]:- "true"
+[_metadata_:components_material]:- "true"
+[_metadata_:components_material_description]:- "a pinch of diamond dust worth 25 gold sprinkled over the target, which the spell consumes"
+[_metadata_:components_material_cost]:- "25 gp"
+[_metadata_:duration]:- "8 hours"
+[_metadata_:concentration]:- "false"
+[_metadata_:compared_to_wotc_srd_5.1]:- "mechanics_same_wording_different"
+[_metadata_:compared_to_a5e_srd]:- "mechanics_different_wording_different"
 <!-- markdownlint-disable-next-line no-emphasis-as-heading -->
 _3rd-level abjuration_
 
 **Casting Time:** 1 action \
 **Range:** Touch \
-**Components:** V, S, M (a pinch of diamond dust worth 25 gp sprinkled over the target, which the spell consumes) \
+**Components:** V, S, M (a pinch of diamond dust worth 25 gold sprinkled over the target, which the spell consumes) \
 **Duration:** 8 hours
 
-A creature, place or object is hidden from divination magic and cannot be perceived by magical scrying sensors.
+A willing creature, or a place or object is hidden from divination magic and cannot be perceived by magical scrying sensors.
```

Comment:
WotC specifically states that it only works on a _willing_ creature. A5e doesn't, which opens it up to... Well, a lot of potential bad stuff.